app/novu/emails/novu-onboarding-email.tsx

```diff
@@ -124,7 +86,7 @@ export const NovuWelcomeEmail = ({
                   {item.componentType === "list" ? (
                       <Column>
                         <ul>
-                          {item.componentListItems?.map((item: any, index: number) => (<li className="mb-20" key={index}><strong>{item.title}</strong> {item.body}</li>))}
+                          {item.componentListItems?.map((item: ListElementComponent, index: number) => (<li className="mb-20" key={index}><strong>{item.title}</strong> {item.body}</li>))}
```

Comment:
```suggestion
                          {item.componentListItems?.map((item: ListElementComponent, listItemIndex: number) => (<li className="mb-20" key={listItemIndex}><strong>{item.title}</strong> {item.body}</li>))}
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Is there something more unique besides `index` we can use for‏ key?

Reply:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** We are using a discriminated union, which means that the `item.componentType === "list"` condition above should discriminate the typing that item is of type `ListElementComponent []` - we shouldn't need to add in-line typings. Let's investigate and fix this, as this type of discrimination will be a common use-case for managed lists of components with different types.
‏
```suggestion
                          {item.componentListItems?.map((listItem, listItemIndex) => (<li className="mb-20" key={index}><strong>{item.title}</strong> {item.body}</li>))}
```

Reply:
renamed index, removed redundant types

@rifont The type was there because I went with adding a concrete type in place of any, the type inferring is a better choice

Reply:
> ❓ question: Is there something more unique besides index we can use for‏ key?

I fail to see a different option, as both title and body in this case could have the same content